src/build/run_xcodebuild.mjs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2023 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {spawnStream} from './spawn_stream.mjs';
+
+const runXcodebuild = (command, flags) =>
```

Comment:
We need a way to pass arbitrary args that are not prefixes with `-`. I'd remove the flag mapping.
There are also situations where the order matters, or you may have duplicate flags.